download-geofabrik.go

```diff
@@ -1,7 +1,7 @@
 package main
 
 import (
-	"crypto/md5"
+	"crypto/md5" //nolint:gosec
```

Comment:
Ignoring gosec because I know md5 is a bad hashing method but I've no choice :smile: